closed/autoconf/custom-hook.m4

```diff
@@ -37,12 +39,35 @@ AC_DEFUN_ONCE([CUSTOM_EARLY_HOOK],
   AC_SUBST(OPENJ9OMR_TOPDIR)
   AC_SUBST(OPENJ9_TOPDIR)
 
+  OPENJ9_LIB_DETERMINE_DEPENDENCIES
+  OPENJ9_LIB_SETUP_LIBRARIES
+  OPENJ9_BASIC_SETUP_FUNDAMENTAL_TOOLS
   OPENJ9_PLATFORM_SETUP
   OPENJDK_VERSION_DETAILS
   OPENJ9_CONFIGURE_CUDA
   OPENJ9_THIRD_PARTY_REQUIREMENTS
 ])
 
+AC_DEFUN([OPENJ9_LIB_DETERMINE_DEPENDENCIES],
+[
+  # Check if numa is needed
+  if test "x$OPENJDK_TARGET_OS" = xlinux && test "x$OPENJDK_BUILD_CPU_ARCH" != xs390; then
```

Comment:
I know we don't have ARM builds active yet, but this condition seems a bit too open ended to me. Perhaps better to explicitly list the platforms where it should be done?

Reply:
Openj9 supports numa on x86 and ppc architectures, so I updated the condition accordingly.